### DISCUSSION TITLE
Xcode issues: method should not be called on the main thread

### DISCUSSION THREAD

**Discussion by TD540** [kylehickinson/SwiftUI-WebView#11] [General]
After importing and using this SwiftUI-WebView package, Xcode 14.1 gives me lots of issues/warnings like this:

`This method should not be called on the main thread as it may lead to UI unresponsiveness.`

Upon clicking these issues, Xcode points me to the @main App struct...
Can this be fixed?

https://imgur.com/a/0DJzc69

**Comment by kylehickinson** [comment]
This is common with Xcode 14 in general and is not related to this package, unfortunately there is no workaround that I know of to stop it from occurring. It seems like it is just bad diagnostics from Apple
